cloud-core/src/main/java/cloud/commandframework/arguments/standard/StringArgument.java

```diff
@@ -309,6 +314,40 @@ public StringParser(
                 }
                 inputQueue.remove();
                 return ArgumentParseResult.success(input);
+            } else if (this.stringMode == StringMode.QUOTED) {
+                final StringJoiner sj = new StringJoiner(" ");
+                for (final String string : inputQueue) {
+                    sj.add(string);
+                }
+                final String string = sj.toString();
+
+                Matcher matcher = QUOTED_DOUBLE.matcher(string);
+                String inner = null;
+                if (matcher.find()) {
+                    inner = matcher.group("inner");
+                } else {
+                    matcher = QUOTED_SINGLE.matcher(string);
+                    if (matcher.find()) {
+                        inner = matcher.group("inner");
+                    }
+                }
+
+                if (inner != null) {
+                    final int numSpaces = (int) inner.chars().filter(c -> c == ' ').count();
+                    for (int i = 0; i <= numSpaces; i++) {
+                        inputQueue.remove();
+                    }
+                } else {
+                    inner = inputQueue.remove();
+                    if (inner.startsWith("\"") || inner.startsWith("'")) {
+                        return ArgumentParseResult.failure(new StringParseException(sj.toString(),
+                                StringMode.QUOTED, commandContext));
+                    }
+                }
+
+                inner = inner.replace("\\\"", "\"").replace("\\\'", "\"");
```

Comment:
this is replacing escaped single quotes with double quotes. needs to be this:
``inner = inner.replace("\\\"", "\"").replace("\\'", "'");``